### PR TITLE
[PyOV] Fix deprecation warning in tests

### PR DESCRIPTION
**Pull request by akuporos**
### Details:
 - Some overloadings of `register_pass` are deprecated and should be removed in this release, so I updated tests
 -  17 warnings -> 0 warnings for `tests` folder.
 -  24 warnings -> 0 warnings for `tests_compatibility` folder.

### Tickets:
 - 104388

### CODE REVIEW

File: src/bindings/python/tests/test_graph/test_manager.py [hunk 1]
```diff
@@ -8,17 +8,26 @@
 import numpy as np
 import pytest
 
-import openvino.runtime.opset8 as ov
-from openvino.runtime import Model
+import openvino.runtime.opset8 as ops
+from openvino.runtime import Core, Model
```
Comment (jiwaszki) [inline src/bindings/python/tests/test_graph/test_manager.py:12]:
```suggestion
import openvino.runtime.opset10 as ops
```
Bump the opset?

**Review by jiwaszki — APPROVED** [review]

File: src/bindings/python/tests/test_graph/test_manager.py (hunk 1 above)
Comment (p-wysocki) [reply src/bindings/python/tests/test_graph/test_manager.py:12]: I don't think we have opset11 yet https://github.com/openvinotoolkit/openvino/tree/master/src/core/include/openvino/opsets

**Review by p-wysocki — APPROVED** [review]

File: src/bindings/python/tests/test_graph/test_manager.py (hunk 1 above)
Comment (akuporos) [reply src/bindings/python/tests/test_graph/test_manager.py:12]: good catch!

File: src/bindings/python/tests/test_graph/test_manager.py [hunk 2]
```diff
@@ -8,17 +8,26 @@
 import numpy as np
 import pytest
 
-import openvino.runtime.opset8 as ov
-from openvino.runtime import Model
+import openvino.runtime.opset8 as ops
```
Comment (akuporos) [inline src/bindings/python/tests/test_graph/test_manager.py:11]:
```suggestion
import openvino.runtime.opset10 as ops
```

**Pull request merged**